lib/resty/core/shdict.lua

```diff
@@ -387,6 +395,79 @@ local function shdict_flush_all(zone)
 end
 
 
+local function shdict_ttl(zone, key)
+    zone = check_zone(zone)
+
+    if key == nil then
+        return nil, "nil key"
+    end
+
+    if type(key) ~= "string" then
+        key = tostring(key)
+    end
+
+    local key_len = #key
+    if key_len == 0 then
+        return nil, "empty key"
+    end
+    if key_len > 65535 then
```

Comment:
Better add a blank line between successive if statement blocks.

Reply:
I certainly concur. It was only like so to be consistent with the above `incr()` method. Fixed.